animal_crossing.py

```diff
@@ -83,9 +164,14 @@ def _stalk_set_sell_price(cmd, price):
 	with db:
 		cur = db.execute('SELECT week_index, price FROM sell_price WHERE user_id = ? AND week_local = ?',
 				(user_id, str(sunday),))
-		week_price_rows = cur.fetchall()
+		week_sell_rows = cur.fetchall()
+		cur = db.execute('SELECT price FROM buy_price WHERE user_id = ? AND week_local = ?',
+				(user_id, str(sunday),))
+		week_buy_row = cur.fetchone()
 	week_prices = [None] * 13
-	for row in week_price_rows:
+	if week_buy_row: 
```

Comment:
```suggestion
	if week_buy_row:
```
sorry lol my change causes this to fail pylint because there's trailing whitespace now